Lecture6/Task1/HttpClientsImplementation/Utils/RequestExecutor.java

```diff
@@ -0,0 +1,45 @@
+package HttpClientsImplementation.Utils;
+
+import Constants.HttpClients;
+import Constants.HttpMethods;
+import HttpClientsImplementation.ApacheHttpClient;
+import HttpClientsImplementation.RestAssured;
+
+public class RequestExecutor {
+
+    private static final String ERROR_MESSAGE = "Something went wrong => RequestExecutor)";
+
+    public RequestExecutor() {
+    }
+
+    @SuppressWarnings("unchecked")
```

Comment:
good way to ignore valuable warnings. Avoid such things, please in your practice. 
I believe compiler show you here not save type cast, right? Make it save without ignoring warnings.

Reply:
fixed
CustomResponse class was added